test/development/acceptance/server-component-compiler-errors-in-pages.test.ts

```diff
@@ -137,10 +141,8 @@ describe('Error Overlay for server components compiler errors in pages', () => {
          4 |   return 'hello world'
            \`----
 
-        Import trace for requested module:
-        ./components/Comp.js
-        ./pages/index.js"
-      `)
+        Import trace for requested module:"
```

Comment:
The exact order of the requested modules doesn't seem deterministic, thus causing flakiness